integration-tests/testconfig/ccip/ccip.toml

```diff
@@ -83,6 +83,87 @@ evm_supports_eip1559 = true
 evm_default_gas_limit = 6000000
 evm_finality_depth = 1
 
+# SIMULATED_1_DEEPER_FINALITY, SIMULATED_2_DEEPER_FINALITY, SIMULATED_3_DEEPER_FINALITY
+# are the same as SIMULATED_1, SIMULATED_2, SIMULATED_3, but with finality_depth = 10.
+# These are primarily used in the reorg tests, but can technically be used in any tests.
+[Network.EVMNetworks.SIMULATED_1_DEEPER_FINALITY]
+evm_name = 'chain-1337'
+evm_chain_id = 1337
+evm_keys = [
+    "59c6995e998f97a5a0044966f0945389dc9e86dae88c7a8412f4603b6b78690d",
+    "ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80",
+    "5de4111afa1a4b94908f83103eb1f1706367c2e68ca870fc3fb9a804cdab365a",
+    "7c852118294e51e653712a81e05800f419141751be58f605c371e15141b007a6",
+    "47e179ec197488593b187f80a00eb0da91f1b9d0b13f8733639f19c30a34926a",
+    "8b3a350cf5c34c9194ca85829a2df0ec3153be0318b5e2d3348e872092edffba",
+    "92db14e403b83dfe3df233f83dfa3a0d7096f21ca9b0d6d6b8d88b2b4ec1564e",
+    "4bbbf85ce3377467afe5d46f804f221813b2bb87f24d81f60f1fcdbf7cbf4356",
+    "dbda1821b80551c9d65939329250298aa3472ba22feea921c0cf5d620ea67b97",
+    "2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6",
+    "f214f2b2cd398c806f84e317254e0f0b801d0643303237d97a22a48e01628897"
+]
+evm_simulated = true
+client_implementation = 'Ethereum'
+evm_chainlink_transaction_limit = 50000
+evm_transaction_timeout = '2m'
+evm_minimum_confirmations = 1
+evm_gas_estimation_buffer = 1000
+evm_supports_eip1559 = true
+evm_default_gas_limit = 6000000
+evm_finality_depth = 10
```

Comment:
nit : is the chainlink node config updated as part of this updated finality? if not you can create a similar func like [this](https://github.com/smartcontractkit/chainlink/blob/develop/integration-tests/types/config/node/core.go#L89) to update the finality in `evmConfig.Chain ` from the finality mentioned in blockchain.EVMNetwork input param. It's not updated in the existing function because other product do not use finality.

Reply:
Oh interesting, yeah I do change the settings of the Chainlink node but its via the TOML overrides, I suppose that's not the best way to do things?